r-package/inst/include/models/hiv_demographic_projection.hpp

```diff
@@ -36,6 +36,7 @@ struct HivDemographicProjection<Config> {
   static constexpr int PROJPERIOD_MIDYEAR = SS::PROJPERIOD_MIDYEAR;
```

Comment:
I think that the calculations done in `run_hc_hiv_pop_end_year_migration()` also need to be implemented where mid-year net migration is done.

Relevant code here: https://github.com/mrc-ide/frogger/blob/736257433520a41b9475658bd94337c320b44929/r-package/inst/include/models/hiv_demographic_projection.hpp#L310-L336

(That is obsolete in Spectrum now; but have it implemented in the adult model, so suggest keeping consistent here)

Note that there it will be `if (t = p_hc.hc_art_start)` rather than `if (t >= p_hc.hc_art_start)`